configs/configs.go

```diff
@@ -2,26 +2,54 @@ package configs
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/LetsFocus/goLF/logger"
 	"github.com/joho/godotenv"
 )
 
-func NewConfig(log *logger.CustomLogger) Config {
-	env := os.Getenv("APP_ENV")
-	envPath := ""
-	if env != "" {
-		envPath = "./configs/." + env + ".env"
-	} else {
-		envPath = "./configs/.env"
+func findConfigsDir(dir string) (string, error) {
```

Comment:
keep this function at the bottom since it is a helper function, let's keep all the helper functions at the bottom